--- ed-1.0/Makefile.in.orig	2008-05-04 17:57:10.000000000 +0200
+++ ed-1.0/Makefile.in	2008-08-23 23:14:21.434113358 +0200
@@ -59,7 +59,7 @@
 	if test ! -d $(DESTDIR)$(mandir)/man1 ; then $(INSTALL) -d $(DESTDIR)$(mandir)/man1 ; fi
 	$(INSTALL_DATA) $(VPATH)/doc/$(progname).1 $(DESTDIR)$(mandir)/man1/$(program_prefix)$(progname).1
 	-rm -f $(DESTDIR)$(mandir)/man1/$(program_prefix)r$(progname).1
-	cd $(DESTDIR)$(mandir)/man1 && ln -s $(program_prefix)$(progname).1 $(program_prefix)r$(progname).1
+	cd $(DESTDIR)$(mandir)/man1 ; echo ".so $(program_prefix)$(progname).1" >$(program_prefix)r$(progname).1
 
 install-strip : all
 	$(MAKE) INSTALL_PROGRAM='$(INSTALL_PROGRAM) -s' install
