--- ed-1.6/Makefile.in.orig	2011-04-16 13:17:15.000000000 +0200
+++ ed-1.6/Makefile.in	2012-01-04 19:47:48.352289420 +0100
@@ -18,10 +18,10 @@
 all : $(progname) r$(progname)
 
 $(progname) : $(objs)
-	$(CC) $(LDFLAGS) -o $@ $(objs)
+	$(CC) $(LDFLAGS) $(CFLAGS) -o $@ $(objs)
 
 $(progname)_profiled : $(objs)
-	$(CC) $(LDFLAGS) -pg -o $@ $(objs)
+	$(CC) $(LDFLAGS) $(CFLAGS) -pg -o $@ $(objs)
 
 r$(progname) : r$(progname).in
 	cat $(VPATH)/r$(progname).in > $@
